DESCRIPTION

```diff
@@ -2,7 +2,15 @@ Package: ggthemr
 Title: Themes for ggplot2
 Description: Colour schemes and layouts for ggplot2.
 Version: 1.1.0
-Author: Ciaran Tobin
+Date: 2020-11-01
+Authors@R: c(
+    person("Ciaran", "Tobin", , "cttobin@hotmail.com", "aut"),
+    person("Amy Tzu-Yu", "Chen", , "amy17519@gmail.com", role = "cre",
+           comment = c(ORCID = "0000-0002-4085-572X")),
+    person("Gergely", "Daróczi", , "daroczig@rapporter.net", role = "aut",
```

Comment:
No need to include me -- I have done nothing yet :zipper_mouth_face:

Reply:
well not exactly nothing. changing to 'ctb'